src/utils/navigatorData.js

```diff
@@ -179,16 +179,21 @@ export function getSiblings(uid, childrenMap, children) {
 }
 
 function extractRootNode(data) {
+  // note: this "root" path won't always necessarily come at the beginning of
+  // the URL in situations where the renderer is being hosted at some path
+  // prefix
+  const rootPathPattern = /^.*(\/documentation\/[^/]+).*$/;
```

Comment:
Anchoring the start & end of the pattern with `^` and `$` is cancelled out by matching `.*` inside the anchor, so we can probably remove both.

Reply:
Is there any way we can do this without assuming that the value of `node.path` a) definitely starts with `/documentation` and b) definitely contains only two path segments? It feels like it’s more likely to be robust against any future changes if we make as few assumptions as possible. (But maybe it’s not possible to avoid either of those!)

Reply:
> Is there any way we can do this without assuming that the value of node.path a) definitely starts with /documentation

Due to my note above in the comment about this path not always necessarily starting at the start of the URL path, I'm not sure there's an alternative I'm aware of.

> b) definitely contains only two path segments?

There isn't any other indicator we could use that I'm aware of. This code is already only executing for the top level items in the outermost (most-top level) array and only when there are multiple of these root nodes.

I'm up for any suggestions if you have any though. I agree that the less assumptions the better, but I'm not sure how this could be made any more generic at the moment.

Reply:
> Anchoring the start & end of the pattern with `^` and `$` is cancelled out by matching `.*` inside the anchor, so we can probably remove both.

Fixed: 8de15f66edc94afd5daba5cf6b28e57ba46375f6